.github/workflows/CI.yml

```diff
@@ -0,0 +1,38 @@
+# CI to build asciidoctor spec targets, on push or manually
+
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
```

Comment:
Is the comment correct here that the workflow is triggered on both "push" and "pull"?  I don't know this yaml file scheme, but since it just says "push:" here, it makes me think the action is triggered only on "push" requests.

Also out of curiosity, why is the workflow limited to just the "master" branch?  I don't see that branch name anywhere.  Is it the default?

Reply:
Good catch, thanks. Those are left over comments from a different branch I copied a starting point from.

Reply:
Good catch, thanks. Those are left over comments from a different branch I copied a starting point from.

Reply:
N.b. gitlab CI YAML is bad but github Actions is worse.